.php_cs.dist

```diff
@@ -15,18 +15,20 @@ $config
         'array_syntax' => array('syntax' => 'long'),
         'binary_operator_spaces' => false,
         'concat_space' => array('spacing' => 'one'),
+        'increment_style' => false,
+        'no_alias_functions' => true,
         'no_useless_else' => true,
         'no_useless_return' => true,
         'ordered_imports' => true,
         'phpdoc_no_package' => false,
         'phpdoc_order' => true,
         'phpdoc_summary' => false,
         'pre_increment' => false,
-        'trailing_comma_in_multiline_array' => false,
         'simplified_null_return' => false,
+        'trailing_comma_in_multiline_array' => false,
```

Comment:
While we're moving it, can we please turn this rule on? A trailing comma is desirable, as it makes git blame more obvious.

Reply:
Sure, can do!